pkg/generator/helper/resources.go

```diff
@@ -230,6 +230,7 @@ func (p *{{ .StepName }}{{ .Name | title}}) persist(stepConfig {{ .StepName }}Op
 	gcsClient, err := gcs.NewClient(gcs.WithEnvVars(envVars))
 	if err != nil {
 		log.Entry().Errorf("creation of GCS client failed: %v", err)
+        return
```

Comment:
```suggestion
		return
```

Reply:
Thanks! Will do - Sunday PR's aren't the best...